renpy/atl.py

```diff
@@ -1183,11 +1183,6 @@ def execute(self, trans, st, state, events):
 
         warper = warpers.get(self.warper, self.warper)
 
-        if trans.atl.animation:
```

Comment:
It's very weird that this statement ends up not being used, while it was before we fixed the ATL fast-forward